tests/integration/src/utils/fluent_bit_manager.py

```
@@ -129,58 +255,163 @@ def stop(self):
+            leaks_binary,
+            "--quiet",
+            "--fullStacks",
+            "--atExit",
+            "--",
+            "/bin/sh",
+            "-c",
+            LEAKS_EXEC_SCRIPT,
+            "fluent-bit-leaks-target",
+            self.leaks_target_pid_file,
+            self.log_file,
+        ] + command
```

Ruff (0.15.21)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)